benchmarks/common/common_ops_test.py

```diff
@@ -2,6 +2,7 @@
 #
 # This source code is licensed under the MIT license found in the
 # LICENSE file in the root directory of this source tree.
+from __future__ import annotations
```

Comment:
does this import need to be added to all files?

Reply:
In general it doesn't hurt and I'm always pissed when python 3.9 tests fail bc it's missing